src/compiler/base/invoke-compiled/invoke-compiled-child-scenarios-or-expects.xsl

```diff
@@ -35,9 +35,20 @@
          </xsl:message>
       </xsl:if>
 
-      <xsl:apply-templates select="$this/element() except $handled-child-vardecls"
-         mode="local:invoke-compiled-scenarios-or-expects">
-         <xsl:with-param name="reason-for-pending" select="$reason-for-pending" tunnel="yes"/>
+      <!-- Generate invocation instructions. Wrap them in a single document so that they have
+         adjacent relationship. -->
+      <xsl:variable name="invocation-doc" as="document-node()">
+         <xsl:document>
+            <xsl:apply-templates select="$this/element() except $handled-child-vardecls"
+               mode="local:invoke-compiled-scenarios-or-expects">
+               <xsl:with-param name="reason-for-pending" select="$reason-for-pending" tunnel="yes" />
+            </xsl:apply-templates>
+         </xsl:document>
+      </xsl:variable>
+
+      <!-- Group the invocation instructions -->
+      <xsl:apply-templates select="$invocation-doc" mode="x:group-invocation">
+         <xsl:with-param name="tunnel_invoker-description-or-scenario" select="." tunnel="yes" />
```

Comment:
What is `$tunnel_invoker-description-or-scenario` for? It doesn't seem to be used.

Reply:
It was used in multi-threading which is not included in this pr. Removed in 775c0f681864a55eda35d4ba06d65376c9495851.